pyembed/src/test/test_importer_resource_collector.py

```diff
@@ -17,6 +21,24 @@
 )
 
 
+@contextmanager
+def assert_tempfile_cleaned_up(TemporaryDirectory=tempfile.TemporaryDirectory):
+    call_count = 0
+    class TrackingTempDir(TemporaryDirectory):
+        def cleanup(self):
+            nonlocal call_count
```

Comment:
TIL there is a `nonlocal` keyword in Python! How, I'm not sure, because `nonlocal` is obviously a better choice for many situations compared to `global`!